src/Compilers/CSharp/Test/Semantic/Semantics/RefEscapingTests.cs

```diff
@@ -163,9 +163,10 @@ void M()
                 Diagnostic(ErrorCode.ERR_LockNeedsReference, "s").WithArguments("System.Span<int>").WithLocation(9, 15));
         }
 
-        [Fact]
-        public void RefStructEscapeInIterator()
-
+        [Theory]
+        [InlineData(LanguageVersion.CSharp10)]
+        [InlineData(LanguageVersion.CSharp11)]
+        public void RefStructEscapeInIterator(LanguageVersion languageVersion)
```

Comment:
Many of the changes in this file are to test with both C#10 and C#11 escape rules. There are a few cases where the results differ though.

Reply:
This works out nicely. Expanded several more tests to use this.